datadotworld/client/api.py

```diff
@@ -307,7 +358,7 @@ def sync_files(self, dataset_key):
         except _swagger.rest.ApiException as e:
             raise RestApiError(cause=e)
 
-    def upload_files(self, dataset_key, files):
+    def upload_files(self, dataset_key, files, **kwargs):
```

Comment:
How can a user define `description` and `labels` for each of the uploaded files?